kubetest/e2e.go

```diff
@@ -48,15 +48,21 @@ func argFields(args, dump, ipRange string) []string {
 }
 
 func run(deploy deployer, o options) error {
+	cmd, err := deploy.KubectlCommand()
+	if err != nil {
+		return err
+	}
+	if cmd == nil {
+		cmd = exec.Command("./cluster/kubectl.sh")
+	}
 	if o.checkSkew {
-		os.Setenv("KUBECTL", "./cluster/kubectl.sh --match-server-version")
-	} else {
-		os.Setenv("KUBECTL", "./cluster/kubectl.sh")
+		cmd.Args = append(cmd.Args, "--match-server-version=false")
```

Comment:
this flag seems really unfortunate @krzyzacy (not due to this PR, just in general :/)

Reply:
is this changing `--match-server-version` to `--match-server-version=false`?

Reply:
Oh let me fix